yoyodyne/models/__init__.py

```diff
@@ -26,6 +29,7 @@ def get_model_cls(arch: str) -> BaseEncoderDecoder:
         "attentive_lstm": AttentiveLSTMEncoderDecoder,
         "lstm": LSTMEncoderDecoder,
         "pointer_generator_lstm": PointerGeneratorLSTMEncoderDecoder,
+        "pointer_generator_transformer": PointerGeneratorTransformerEncoderDecoder,
```

Comment:
Flake8 complains about this. If I put the model on a new line, black complains. @kylebgorman do you know the solution here?

Reply:
Just use a `  # noqa` on the offending lines.